tools/cargo-wrap.sh

```
@@ -0,0 +1,167 @@
+      need_check="false"
+      break
+    fi
+  done
+  if [[ "$need_check" == "true" ]]; then
+    ARGS+=("--" "--check")
+  fi
+fi
+
+# Helpers
+count_warnings() {
+  grep "^warning:" "$TMP" | grep -v "generated.*warning" | wc -l | tr -d ' '
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)